lib/encoder.py

```diff
@@ -766,7 +770,11 @@ def _FilesToEncodings(self, files, videofile, bitrate=None,
         encoder or self._FileNameToEncoder(full_filename),
         bitrate or _FileNameToBitrate(full_filename),
         videofile or _FileNameToVideofile(full_filename))
-      candidate.Recover()
+      try:
+        candidate.Recover()
+      except Error:
```

Comment:
I discovered that some of my files had parameters outside the currently permitted range (libvpx had changed the permitted range of the --cpu-usage parameter). This ignores those files, so that the "best" candidate is guaranteed to be a legal candidate. Yes, there should be a test for this, and "print" in a library is not a good error reporting mechanism.